test/gtest/ucp/test_ucp_request.cc

```diff
@@ -100,3 +102,398 @@ UCS_TEST_P(test_ucp_request, test_request_query)
 }
 
 UCP_INSTANTIATE_TEST_CASE_TLS(test_ucp_request, all, "all")
+
+class test_proto_reset : public ucp_test {
+public:
+    enum operation_e {
```

Comment:
Anonymous ? -> `enum { ... };`
Names? -> `typedef enum { ... } X_t;`

Reply:
+ Prefix name for members

Reply:
class name is prefix (it's c++).